README.md

```diff
@@ -3,39 +3,83 @@
 ![Shakespeare](http://www.gomoxie.com/wp-content/uploads/Hamlet.jpg)
 >To be, or not to be...
 
-## Learning Objectives - The student should be able to..
+## Objectives
 
-* Explain that by making parts of their code *conditional*, they can execute different pieces of code based on certain conditions.
-* Explain that an *if* statement will execute a set of statements **only** if that condition is *true*. 
+1. Describe the meaning of *conditional statements*
+1. Use the `if` statement to conditionally execute a set of statements
+1. Use an `else` clause to provide an alternative set of statements
+1. Chain multiple *if-else* statements together
+
+## Conditional statements
+
+You already learned how to use the `Bool` type and various operators to evaluate logical expressions and compare values:
 
 ```swift
 var age = 19
 
+let isOfDrinkingAge = age > 21 // false, because 19 is not greater than 21
+```
+
+As was briefly mentioned in that lesson, those expressions become powerful using *conditional statements* which allow you to write pieces of code which can be executed (or not executed) based on certain conditions. These are called *control flow* constructs, because they allow you to control the execution flow of your program, allowing your program's output to react to conditions, instead of always providing the same output.
+
```

Comment:
In general, we usually don't reference prior lessons so I'm more inclined to take that part out to allow for the reading to stand alone.

I'm more inclined to first define what a *control flow* construct is and humanize it so that way the student can see very clearly the problem they are trying to solve and how they can use this new found knowledge. Stating something like.. "Control flow lets you tell your program what code to execute conditionally. As humans, we actually enact flow control every day. For instance, if you are hungry, you will go and get a snack. Otherwise, you'll stay put and continue to read this awesome readme. In the context of an application, for example, you can easily think of content or functionality on a view you've visited that is only available to a user if that user is logged in."